src/pages/index.js

```diff
@@ -104,3 +104,31 @@ addCardButton.addEventListener("click", () => {
   addFormValidator.toggleButtonState();
   newFormPopup.open();
 });
+
+fetch("https://around-api.en.tripleten-services.com/v1", {
```

Comment:
Needs correction: please remove lines 108-116, you should fetch api only via Api class